### DISCUSSION TITLE
Tax Tools and Chicken withdrawals

### DISCUSSION THREAD

**Discussion by Polysemouse** [sunflower-land/sunflower-land#1527] [General]
**Problem**
 
Currently there are no 30% withdraw tax for tools (axes, all sorts of pickaxes) and chicken.  For players that want to evade the withdraw tax for SFL, they can do the following:

1. Buy tools using SFL (eg. axe)
2. Withdraw axe
3. Swap on NiftySwap for roughly the same value in SFL (https://beta.niftyswap.io/collectible/polygon/0x22d5f9b75c524fec1d6619787e582644cd4d7422/301?exchange=0x4e229569f61b8917d2b62a84715fffc614312084)

**Suggestion**

There are a few solution to this:
1. Disable withdraw for tools and chickens, like how seeds withdraws are disabled
2. Apply 30% tax on withdrawing tools and chickens, to make it in line with crops and resources
3. Apply 30% tax on withdrawing tools and chickens, to make it in line with crops and resources, but round up the tax (so an integer amount is always withdrawn.  Minimum withdraw amount is 2:
  a. player tries to withdraw 2 axe, they are taxed 1 axe (30% is 0.6 axe).  
  b. player tries to withdraw 20 axe, they are taxed 6 axe (30% is 6 axe).  
  c. player tries to withdraw 21 axe, they are taxed 7 axe (30% is 6.3 axe).

**Comment by SuperSynapse** [comment]
IMO a best first measure would be to simply remove the ability to withdraw tools until the farm is upgraded.

I'm personally not a big fan of a lot of these taxes, especially for dedicated users who are active and have shown a level of investment in the game.

Taxes should be targeted toward low level, low investment accounts, but with paths to remove the majority (if not all) of them as you advance. Incentivising trading, incentivising community interactions.

As a good example, I've recommended SFL to several friends and sent them a few supplies to get started. Simply giving a friend 20 axes and a small collection of pickaxes, etc is a HUGE help especially with how difficult it is to earn SFL from a low level farm currently.


My recommendation:
 - Add taxes as with all items
 - Add level requirements (upgraded farm) to reduce all taxes slightly (30% to 20% with fully upgraded, which in LE will be difficult)
 - Add a path to further reduce that amount with the VIP NFTs (20% down to 10%)
 - Allow role based NFT holders to capitalize on those NFT boosts for a healthy game economy.
 
 People with fully upgraded farms are your REAL players, and should be able to sell if they want (most of them don't), but if we are Play to Own, this is 100% inline with that vision if they choose to.
 
 People with VIP NFTs are your buyers. Although a few may sell based on a specific NFT Boost (chicken coop selling eggs for instance), as a group they sell far less than the normal user, and usually if they do, it's so they can reinvest into other resources/NFTs. The intent was never for someone with a chicken coop to use ALL of the eggs they produce, it was for them to specialize in that role for the larger economy.

**Reply by adamhannigan** [reply]
@SuperSynapse Some great ideas 💪 

The first point of call will be disabling withdrawal of tools - it isn't exactly something that needs to be traded amongst players as it is extremely accessible to mint these in-game.

We are also exploring level-gating and reducing tax for 'loyal' players who aren't extractors in the system

**Reply by future2back** [reply]
The first point: Shouldn't this be considered abuse of game glitch and players punished? Thank you.

**Reply by Myth727** [reply]
I would not call that a game glitch,  users just playing and seeing they have the ability to withdraw tools do what they can. The team can see large amounts of tools bought and withdrawn daily. Let's not aim to punish everyone haha

**Comment by AL-Simmons92** [comment]
Nice little loop hole you discovered. Good Job. Great Community wins again

**Comment by NanndoSnowzera** [comment]
I have an idea in development that was for something else, but it could come down well here, I'll plan this to be one of the ways to try to help "relieves" this fee for loyal players who don't just want to extract, when they serve or want exchanging something most of the time is to reinvest in another new NFT or something along those lines. I have a proposal that I will soon release to be analyzed.
